apps/chrome-devtools/src/services/localization.service.ts

```diff
@@ -0,0 +1,85 @@
+import { effect, inject, Injectable, signal, type Signal } from '@angular/core';
+import { toObservable, toSignal } from '@angular/core/rxjs-interop';
+import type {
+  GetTranslationValuesContentMessage,
+  IsTranslationDeactivationEnabledContentMessage,
+  LanguagesContentMessage,
+  LocalizationsContentMessage,
+  SwitchLanguageContentMessage
+} from '@o3r/localization';
+import { distinctUntilChanged, filter, map, switchMap } from 'rxjs/operators';
+import { ChromeExtensionConnectionService, filterAndMapMessage } from './connection.service';
+
+@Injectable({ providedIn: 'root' })
+export class LocalizationService {
+  private readonly connectionService = inject(ChromeExtensionConnectionService);
+  public readonly localizationsMetadata = toSignal(
+    this.connectionService.message$.pipe(
+      filterAndMapMessage(
+        (message): message is LocalizationsContentMessage => message.dataType === 'localizations',
+        (message) => message.localizations
+      )
+    ),
+    { initialValue: [] }
+  );
+  public readonly languages = toSignal(
+    this.connectionService.message$.pipe(
+      filterAndMapMessage(
+        (message): message is LanguagesContentMessage => message.dataType === 'languages',
+        (message) => message.languages
+      )
+    ),
+    { initialValue: [] }
+  );
+  public readonly isTranslationDeactivationEnabled = toSignal(
+    this.connectionService.message$.pipe(
+      filterAndMapMessage(
+        (message): message is IsTranslationDeactivationEnabledContentMessage => message.dataType === 'isTranslationDeactivationEnabled',
+        (message) => message.enabled
+      )
+    ),
+    { initialValue: false }
+  );
+  private readonly lang = signal<string | undefined>(undefined);
```

Comment:
Should be declared before `public` field.
I was thinking the linter would check it :S